### DISCUSSION TITLE
Add imagemagick dart api url to the develop list on the imagemagick website

### DISCUSSION THREAD

**Discussion by Haidar0096** [ImageMagick/ImageMagick#7267] [Contribute]
Hello, I created a while ago an image magick plugin for dart that exposes the C api in dart, I noticed that it is not listed on the [website](https://imagemagick.org/script/develop.php), can you please add it so users who come to this page can find it as a reference?
Here are the links (I created 4 packages, same code, but difference is in the precision of the image editing):
- [image_magick_q8 1.0.0](https://pub.dev/packages/image_magick_q8) (Pick this as the link to show on your page if only 1 option is allowed)
- [image_magick_q8_hdri 1.0.0](https://pub.dev/packages/image_magick_q8_hdri)
- [image_magick_q16 1.0.0 ](https://pub.dev/packages/image_magick_q16)
- [image_magick_q16_hdri 1.0.0 ](https://pub.dev/packages/image_magick_q16_hdri)

**Comment by dlemstra** [comment]
Feel free to create a pull request for our website repository (https://github.com/ImageMagick/Website) where you add this.

p.s. q8-hdri is a really strange combination. I personally would not publish this because nobody will probably use this. Limiting a hdri build to 256 colors is strange when you are using a float (32bits) for each pixel.
